From 72483458cc79161e39a8ac10e99fd24badb0b5db Mon Sep 17 00:00:00 2001
From: Helge Deller <deller@gmx.de>
Date: Thu, 4 Jan 2024 18:50:50 +0100
Subject: [PATCH 23/23] parisc: SEABIOS_HPPA_VERSION 14

Fixes and enhancements:
- Fix 32-bit HP-UX crash (fix in PDC_FIND_MODULE call)
- Fix NetBSD boot (power button fix and add option to disable it)
- Fix FPU detection on NetBSD
- Add MEMORY_HPA module on B160L
- Fix detection of mptsas and esp scsi controllers
- Fix terminate DMA transfer in esp driver (Mark Cave-Ayland)
- Allow booting from esp controller
- initial fixes to allow building a 64-bit SeaBIOS-hppa firmware

Signed-off-by: Helge Deller <deller@gmx.de>
---
 src/parisc/hppa.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/roms/seabios-hppa/src/parisc/hppa.h b/roms/seabios-hppa/src/parisc/hppa.h
index 3b7e7bb5..2a3cab9a 100644
--- a/roms/seabios-hppa/src/parisc/hppa.h
+++ b/roms/seabios-hppa/src/parisc/hppa.h
@@ -3,8 +3,8 @@
 /* this file is included by x86.h */
 
 /* version number of this SeaBIOS firmware */
-#define SEABIOS_HPPA_VERSION 13
-#define SEABIOS_HPPA_VERSION_STR  "13"
+#define SEABIOS_HPPA_VERSION 14
+#define SEABIOS_HPPA_VERSION_STR  "14"
 
 #ifdef __LP64__
 #define CONFIG_64BIT
-- 
2.39.2

